config/convert2rhel.ini

```diff
@@ -11,3 +11,14 @@
 # password       = <insert_password>
 # activation_key = <insert_activation_key>
 # org            = <insert_org>
+
+[host_metering]
+# configure_host_metering          = "0"
+
+[inhibitor_overrides]
+# incomplete_rollback              = "0"
```

Comment:
Also, I think that c2r does not care what value is present in the env var (it is enough just to be set to any value), but I think we should use the value "1" in the examples, as it can be more intuitive. WDYT?
```suggestion
# incomplete_rollback              = "1"
```

Reply:
We could always change it to need a truthy value, i.e. `1`, `true`, `yes`

Reply:
I'm inclined to change this to a boolean value to have it forced in convert2rhel to operate correctly from now on that this is set. 

I'm doing a minimal conversion from "0" and "1" to true/false, but I don't like that solution that much... I will see about converting them to boolean values when reading the config

Reply:
Okay, changed the code to parse the options from inhibitor_overrides as booleans directly. 

From the configparse docs it seems that the accepted values are `on/off, yes/no, 1/0, true/false`. We should be safer as any of those get converted to bool right away and we will be able to enforce it properly in the codebase later.

Reply:
If the tests pass now, I'm ok with merging it